Mobile/starter_project_dartsmiths/lib/features/example/presentation/widgets/aboutMe.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import './status.dart';
+
+import '../../data/datasources/appColor.dart';
+
+class AboutMe extends StatelessWidget {
+  const AboutMe({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(alignment: Alignment.bottomCenter, children: [
+      Padding(
+        padding: const EdgeInsets.all(40),
+        child: Container(
+          decoration: BoxDecoration(
+              color: Colors.white, borderRadius: BorderRadius.circular(13)),
+          height: 300,
```

Comment:
Make use of MediaQuery for getting the height and width of the device and then do the math.